components/01_atoms/Stat2.tsx

```diff
@@ -0,0 +1,10 @@
+export default function Stat2({ description, value, color = "white" }) {
+  const textColorClass = `text-${color}`;
+
+  return (
+    <div className="flex h-28 w-48 max-w-sm flex-col items-center justify-center space-y-1 rounded-xl">
```

Comment:
Instead you could just pass `className` as prop and do that
```suggestion
    <div className={cx("flex h-28 w-48 max-w-sm flex-col items-center justify-center space-y-1 rounded-xl", className)}>
```